upup/models/cloudup/resources/addons/certmanager.io/k8s-1.16.yaml.template

```diff
@@ -5349,26 +5371,27 @@ spec:
             protocol: TCP
           securityContext:
             allowPrivilegeEscalation: false
+            capabilities:
+              drop:
+              - ALL
           env:
           - name: POD_NAMESPACE
             valueFrom:
               fieldRef:
                 fieldPath: metadata.namespace
-          resources:
-            {}
```

Comment:
I think this is safe now that we are using a new approach to applying manifests.